### PR TITLE
Dupli objects support

### PR DESCRIPTION
**Pull request by est77**
- Add support for duplis
- Motion blur related fixes
- Code cleanups

### CODE REVIEW

**Comment by jdent02** [comment]
I'm going to go ahead and merge this as is.  We can fix the interactive update later.

File: translators/dupli.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
+#
```
Comment (dictoon) [inline translators/dupli.py]: Start files with an empty line.

File: translators/dupli.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+# This source file is part of appleseed.
+# Visit http://appleseedhq.net/ for additional information and resources.
```
Comment (dictoon) [inline translators/dupli.py]: The URL should start with `https://`. Make sure this is the case for all files.

File: translators/scene.py [hunk 3]
```diff
@@ -508,9 +519,12 @@ def __calc_motion_subframes(self):
         self.bl_scene.frame_set(current_frame)
 
     def __get_subframes(self, shutter_length, samples):
+        assert(samples > 1)
```
Comment (dictoon) [inline translators/scene.py]: `assert` in Python is an operator, not a function:

    assert samples > 1

File: translators/camera.py [hunk 4]
```diff
@@ -209,11 +209,19 @@ def __thin_lens_camera_params(self, scene, aspect_ratio, film_width, film_height
 
 class InteractiveCameraTranslator(Translator):
 
+    #
```
Comment (dictoon) [inline translators/camera.py]: As much as I like these comments, they might not play nice with autopep8. I would suggest to switch to regular docstrings.

**Pull request merged**